README.md

```diff
@@ -207,6 +208,7 @@ We moved the Bounded Contexts into their own repositories:
 	* `UseCases/`: one directory per use case
 	* `DataAccess/`: implementations of services that binds to database, network, etc
 	* `Infrastructure/`: implementations of services binding to cross cutting concerns, ie logging
+* `contexts/$ContextName`: Concrete implementations of interfaces from the context.
```

Comment:
@JeroenDeDauw What do you think about moving the few remaining implementations out of `contexts` and into `src`? Then we also could get rid of [that confusing line in composer.json where we mess with the PSR-4 loading for [contexts.](https://github.com/wmde/FundraisingFrontend/pull/1239/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R118)

Regardless of the outcome of this question, this should be done in a followup PR, this one can be merged.